src/afl-tmin.c

```diff
@@ -82,6 +82,8 @@ static u8 crash_mode,                  /* Crash-centric mode?               */
     remove_shm = 1,                    /* remove shmem on exit?             */
     debug;                             /* debug mode                        */
 
+static u32 del_len_limit;              /* Minimum block deletion length     */
```

Comment:
should be initialized with 1 as otherwise it is 0?

Reply:
Aha. I think it's safer to initialize it 1.